src/breaking-changes/v-model.md

```diff
@@ -144,7 +144,7 @@ In addition to 2.x hard-coded `v-model` modifiers like `.trim`, now 3.x supports
 <ChildComponent v-model.capitalize="pageTitle" />
 ```
 
-Read more about custom `v-model` modifiers in the [Component Events](http://vuejs.org/guide/components/events.html#usage-with-v-model) section.
+Read more about [custom `v-model` modifiers](https://vuejs.org/guide/components/v-model.html#handling-v-model-modifiers).
```

Comment:
```suggestion
Read more about [custom `v-model` modifiers on components](https://vuejs.org/guide/components/v-model.html#handling-v-model-modifiers).
```

BTW, I feel here we still better mention it's only for component events since the support of forms is another thing here: https://vuejs.org/guide/essentials/forms.html#modifiers

WDYT?

Reply:
@Jinjiang
It's better! I will change it so.

Is the Chinese version "组件上的自定义 `v-model` 修饰符"?
(using DeepL translator)

Reply:
Yes, that's perfect! Let me do this.